src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.CompilationTracker_Generators.cs

```diff
@@ -213,7 +214,7 @@ private partial class CompilationTracker : ICompilationTracker
         }
 
         private async Task<(Compilation compilationWithGeneratedFiles, CompilationTrackerGeneratorInfo generatorInfo)> ComputeNewGeneratorInfoInCurrentProcessAsync(
-            SolutionState solution,
+            SolutionCompilationState compilationState,
```

Comment:
Seems like this should be taking just the SolutionState, right?

Reply:
Will do in followup.

Reply:
Took care of in this PR.